core/market/src/matcher/handlers.rs

```diff
@@ -46,10 +45,7 @@ pub(super) async fn receive_remote_offers(
                         resolver.receive(&offer);
                         offer.id
                     })
-                    .map_err(|e| {
-                        log::info!("Failed to save Offer [{}]. Error: {}", &offer_id, &e);
-                        e
-                    })
+                    .map_err(|e| log::info!("Skipping foreign Offer: {}", e))
```

Comment:
I'm only changing wording here, to change sentiment. This is not a failure, just a notification. This is completely all right to skip some offers.

Reply:
This is related to #1087